mentat/terminal/client.py

```diff
@@ -56,14 +57,16 @@ async def _cprint_session_stream(self):
         async for message in self.session.stream.listen():
             print_stream_message(message)
 
+    async def _handle_loading_messages(self):
+        loading_handler = LoadingHandler()
+        async for message in self.session.stream.listen("loading"):
+            loading_handler.update(message)
+
     async def _handle_input_requests(self):
         while True:
             input_request_message = await self.session.stream.recv("input_request")
             # TODO: Make extra kwargs like plain constants
-            if (
-                input_request_message.extra is not None
-                and input_request_message.extra.get("plain")
-            ):
+            if "plain" in input_request_message.extra:
```

Comment:
Looks like plain is always in `input_request_message` but is usually set to false. I'd revert this change.

Reply:
Just noticed it's @PCSwingle change